pkgs/applications/misc/blender/default.nix

```diff
@@ -80,6 +81,16 @@ stdenv.mkDerivation rec {
         --prefix PYTHONPATH : ${pythonPackages.numpy}/${python.sitePackages}
     '';
 
+  # Set RUNPATH so that libcuda and libnvrtc in /run/opengl-driver(-32)/lib can be
```

Comment:
Maybe this can be abstracted into a hook? The code is very similar. I think we could do with `patchelf`ing all stuff in `/bin` and `/lib` in it.

Reply:
Is it really a good idea to add that RPATH entry to all binaries (once you include the hook)? I remember being bothered by some other hooks, like rpath stripping, not being configurable in that respect.